### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/conversationlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#180]
```
some correction in translation ITA
```

Original issue reported on code.google.com by `liogiu@gmail.com` on 11 Apr 2011 at 1:44

Attachments:
- [conversationlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-180/comment-0/conversationlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Great, thanks you very much liogiu! Please note any more corrections that you 
can spot on the translation.

Committed in r67.
```

Original comment by `oskar.wi...@gmail.com` on 11 Apr 2011 at 6:36
- Changed state: **Ready**
- Added labels: **Type-Enhancement**, **Milestone-0.6.9**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Ok, i will continue the revision and inform you in code.google

Thank to you, great game i'm spending funny time
Giulio
Il giorno 11/apr/2011 20.37, <andors-trail@googlecode.com> ha scritto:
```

Original comment by `liogiu@gmail.com` on 11 Apr 2011 at 7:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 4 May 2011 at 5:39
- Added labels: **Type-Translation**
- Removed labels: **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:51
- Changed state: **Done**
